subscriber/templates/clusterrole.yaml

```diff
@@ -0,0 +1,520 @@
+{{ if .Values.global.clusterScope.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: upgrade-agent-cr
```

Comment:
can we put this name in tpl/user input? This name is common may be a conflict with other tech stack